libs/ui/src/components/molecules/CollapsibleCard/CollapsibleCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { CollapsibleProps } from '@radix-ui/react-collapsible';
+
+import {
+  StyledCollapsibleCard,
+  StyledCollapsibleCardTrigger,
+  StyledCollapsibleCardButton,
+  StyledCollapsibleContent,
+  InnerCard,
+} from './CollapsibleCard.styles';
+import { RiArrowDropDownLine } from 'react-icons/ri';
+
+export type CardProps = CollapsibleProps & {
+  width: string;
+  collapsibleContent?: React.ReactNode;
+};
+
+export const CollapsibleContent = StyledCollapsibleContent;
+
+export const CollapsibleCard = ({
+  open,
+  onChange,
+  width = 'fit-content',
+  children,
+  collapsibleContent,
+}: CardProps) => {
+  return (
+    <StyledCollapsibleCard open={open} onChange={onChange} width={width}>
+      <InnerCard width={'90%'}>
+        {children}
+        <StyledCollapsibleCardTrigger asChild>
```

Comment:
is `asChild` something in our library or is that coming from radix?

Reply:
asChild is implemented by radix. It allows for us to override the element radix uses. In this example redux uses buttons for triggers. We could use a div or SVG or whatever. We use a button but I don't want a button inside a button or to restyle the redux trigger button.